README.md

```diff
@@ -1,36 +1,28 @@
 # ExtJsPackages <a href="https://travis-ci.org/mayflower/ExtJsPackages/"><img src="https://api.travis-ci.org/mayflower/ExtJsPackages.svg" alt="Build Status" style="max-width:100%;"></a>
 ExtJS Packages by Mayflower
 
-## Requirements
+## Using these packages in your application
 
-* ExtJS-5.1.1 [direct download link](http://cdn.sencha.com/ext/gpl/ext-5.1.1-gpl.zip)
-* [Sencha Cmd](http://www.sencha.com/products/sencha-cmd/)
-* [JSDuck](https://github.com/senchalabs/jsduck)
-* [NodeJS](https://nodejs.org/)
-* [npm](https://www.npmjs.com/)
-
-## Setup & Build
-
-1. Go to the `ExtJsPackages` directory.
-```sh
-    $ cd <path_to_workspace>/ExtJsPackages
+* Add the repository
 ```
-2. Install required node packages with npm:
-
-```sh
-    $ npm install
+$ sencha repository add mayflower http://mayflower.github.io/ExtJsPackages
+Sencha Cmd v5.1.3.61
+[INF] Adding remote "mayflower"
+[INF] Remote "mayflower" added
 ```
-3. [Download](http://cdn.sencha.com/ext/gpl/ext-5.1.1-gpl.zip) and extract the ExtJS framework files into the `ext` directory:
-
-```sh
-    $ unzip <path_to>/ext-5.1.1-gpl.zip
-    $ mv ext-5.1.1 ext
+* Embed the packages
+In your app.js adjust the requires array, e.g.:
```

Comment:
A fixed font is required for file and directory names. Putting single backticks around app.js should do the job.